### DISCUSSION TITLE
请问会提供其他平台的运行程序吗

### DISCUSSION THREAD

**Discussion by NaughtyEric** [Meloong-Git/PCL#2427] [提问] [answered]
比如arm/amd芯片的Mac、Linux

**Comment by zkitefly** [comment, marked as answer]
https://github.com/Hex-Dragon/PCL2/issues/54

由于 PCL2 的底层界面框架（WPF）无法跨系统，要兼容其他系统无异于完全重做，不会制作。

**Reply by Cast235** [reply]
最近发布的一个工具箱应该能减小开发量“Game Pointing Toolkit”不知道能不能帮助兼容mac电脑，详见http://b23.tv/1bWAewz （本人不太懂这些，不知道是不是这样的）

**Reply by Chiloven945** [reply]
Java版本身就支持mac以及Linux，不支持的是PCL本身的架构，WPF是专门为Windows开发的你怎么迁移到mac

**Reply by so1ve** [reply]
wine :D

**Reply by Chiloven945** [reply]
你说得对，但是（

**Reply by iPodtouch6557** [reply]
#54 下有讨论好像

**Reply by RyanSon1002** [reply]
悲

**Comment by SuoHouDaiShi** [comment]
你可以试试`wine`之类的玩意，听说管用

**Comment by zkitefly** [comment]
> 你可以试试`wine`之类的玩意，听说管用

不管用，除非虚拟机
